docs/sdk_performance.md

```diff
@@ -0,0 +1,44 @@
+# SDK Performance
+
+## SDK Metrics
+
+1. **The SDK size (723 KB)**. Was measured as the **.aar** package size 
+without the transitive dependencies.For now the SDK uses the following transitive dependencies: 
+     -   androidx.core:core
+     -   androidx.navigation:navigation-fragment-ktx
+     -   androidx.navigation:navigation-ui-ktx
+     -   androidx.navigation:navigation-runtime-ktx
+     -   androidx.recyclerview:recyclerview
+     -   androidx.work:work-runtime
+     -   com.google.code.gson:gson
+     -   com.lyft.kronos:kronos-android
+     -   com.squareup.okhttp3:okhttp
+     -   org.jetbrains.kotlin:kotlin-stdlib
+
+## FAQ
+
+1. Q: **What happens when my application goes in background ?**   
+A: When the application goes in background:
+    -   the auto - instrumentation stops and
+detaches itself from any UI callbacks (lifecycle events, gesture detection). No RUM event will be automatically tracked
+unless it is manually sent through the Public API.
```

Comment:
`unless` doesn't really work here, cause manual and automatic are already excluding one another. I'd say something along the lines of 
> but you can still manually send RUM Events with the RumMonitor.